# 1. Read the research article:

**Functional annotation of human long noncoding RNAs via molecular
phenotyping**

Ramilowski et al. *Genome Research,* 2020.

# 2a. Medical relevant insight

This publication is part of the FANTOM6 project in RIKEN, which focuses
on studying the function of long non-coding RNAs (lncRNAs) in the human
genome.

The study knocked down 285 lncRNAs in human dermal fibroblasts and
identified lncRNAs that, when depleted, alter transcriptional programs
and cellular behaviors (proliferation, morphology, senescence-like
signatures).

The authors then highlight the functional roles for two lncRNAs,
ZNF213-AS1 and lnc-KHDC3L-2, of which the depletion resulted to
transcriptional changes that link to biological pathways and processes
frequently implicated in diseases, such as cancer.

The paper also provides a public lncRNA knockdown dataset with molecular
phenotyping that enables linking lncRNAs to disease-associated genes for
further downstream analysis.

# 2b. Which genomic technologies were used?

Main experimental and molecular approaches in the paper:

-   **RNA knockdown methods** : shRNA and antisense oligonucleotides
    (ASOs) to reduce lncRNA levels; they cross-validated effects across
    reagents.

-   **CAGE (Capped Analysis of Gene Expression)** : deep CAGE sequencing
    to quantify promoter/CAGE tag activity and transcriptome response
    after lncRNA knockdown.

-   **Chromosome conformation capture (Hi**-C) : to predict the
    cis-regulatory function of the target lncRNA to neighboring genes
    and lncRNAs.

-   **Cellular phenotyping** : wound healing assay, cell proliferation
    assay and morphological profiling to capture visible/cellular
    phenotypes.

-   **Computational analyses** : differential expression from CAGE,
    coexpression/pathway enrichment analyses, and clustering to connect
    lncRNAs to pathways and phenotypes.

# 3a. Three questions for further analysis

<u>**Q1. Are the molecular effects of lncRNA knockdown cell-type
specific?**</u>

Many lncRNAs express cell-type specific regulatory roles. In the paper,
the authors used fibroblasts, which revealed some functions, but
additional targets might emerge in other cell lines, for example, stem
cells.

<u>**Q2. lncRNAs regulate transcription via cis/trans manner?**</u>

Knockdow lncRNAs resulted in transcriptional change. How lncRNAs
regulate the transcription of their target genes in term of cis (via
promoter/enhancer modulation to regulate nearby genes) or trans (by
interacting with chromatin modifiers or transcription factors to
regulate distant genes)

<u>**Q3. Do lncRNA knockdowns produce heterogeneous transcription stats
that bulk CAGE misses?**</u>

Single-cell profiling after knockdown will reveal heterogeneous
responses.
